capstone/src/main/java/com/google/step/data/Restaurant.java

```diff
@@ -76,7 +76,7 @@ public String getStory() {
    * @return List<String> cuisine, list of cuisines for the restaurant
    */
   public List<String> getCuisine() {
-    return cuisine;
+    return this.cuisine;
```

Comment:
Its fine to leave it without the 'this'.

Reply:
Yeah, sorry. I was mostly doing this for the sake of consistency. All the other fields have a `this.field`, and it seemed strange that only the `cuisine` didn't.

Reply:
Ah alright.